mzLib/Test/TestDigestionMotif.cs

```diff
@@ -595,96 +593,33 @@ public static void TestProteolysisBothTermini()
         }
 
         [Test]
-        public static void TestAddBiomarkersIntactOnly()
-        {
-            //Note: existing proteoloysis products remain on the list with no additional proteolysis.
```

Comment:
Do you have any thoughts on these tests that were deleted? Were they not helpful regarding this change?

Reply:
intact biomarkers were added to deal w/ methionine cleavage. these are now done at run time in mm.